src/tiled/noeditorwidget.cpp

```diff
@@ -31,9 +32,9 @@ NoEditorWidget::NoEditorWidget(QWidget *parent) :
     ui(new Ui::NoEditorWidget)
 {
     ui->setupUi(this);
-
     connect(ui->newMapButton, &QPushButton::clicked, this, &NoEditorWidget::newMap);
     connect(ui->newTilesetButton, &QPushButton::clicked, this, &NoEditorWidget::newTileset);
+    connect(ui->openFileButton, &QPushButton::clicked, this, &NoEditorWidget::openFile);
```

Comment:
You can connect to this slot directly without needing `NoEditorWidget::openFile`:
```c++
connect(ui->openFileButton, &QPushButton::clicked, DocumentManager::instance(), &DocumentManager::openFile);
```

Reply:
I knew about that one, but I wanted to keep the style of it. I'll follow that way then.